ee/tabby-webserver/src/schema/auth.rs

```diff
@@ -297,6 +313,46 @@ pub struct RequestInvitationInput {
     pub email: String,
 }
 
+#[derive(Validate, GraphQLInputObject)]
+pub struct RequestPasswordResetInput {
+    #[validate(email(code = "email"))]
+    pub email: String,
+}
+
+#[derive(Validate, GraphQLInputObject)]
+pub struct PasswordResetInput {
+    #[validate(email(code = "email"))]
+    pub email: String,
```

Comment:
```suggestion
```

Email shouldn't be required for password reset (it's a token authorize the user_id anyway)

Reply:
That's not true; the user cannot be authorized if they are resetting their password. It needs to be an unauthenticated endpoint.

Reply:
this is more of a best practice. Please implement it as I described. (it's also the way django-auth-graphql does)